fog/ingest/server/src/config.rs

```diff
@@ -38,7 +38,7 @@ pub struct IngestConfig {
     pub peer_listen_uri: IngestPeerUri,
 
     /// List of all peers in this cluster
-    #[structopt(long)]
+    #[structopt(long, use_delimiter = true)]
```

Comment:
Why was this needed?

Reply:
if you want to have it work like `--peers igp://foo,igp://bar,igp://baz,igp://qaz` you need to put `use_delimter = true`. Otherwise, when structopt sees a Vec, you must make it like `--peers igp://foo --peers igp://baz --peers igp://qaz` and that just seems harder to read

Reply:
here's the issue in structopt repo: https://github.com/TeXitoi/structopt/issues/94